index.js

```diff
@@ -14,17 +14,29 @@ const {apiResolver} = (() => {
 const { inspect } = require('node:util')
 
 const rootPath = path.resolve('.').replace(/\\/g, '/');
-const nextPagesDirectory = fs.existsSync(`${rootPath}/pages`) ? `${rootPath}/pages` : `${rootPath}/src/pages`;
+const { pageExtensions } = require(`${rootPath}/next.config.js`);
```

Comment:
@heldaolima FYI we're seeing this change break our projects, raised an issue here https://github.com/mtbrault/nextjs-http-supertest/issues/19